images/eventlistenersink/patches/sink.patch

```diff
@@ -15,7 +15,7 @@ index c6d63bfb..633b75a2 100644
  
  		// TODO: Plumb through context from EL
 -		interceptorResponse, err := interceptors.Execute(context.Background(), r.HTTPClient, &request, url.String())
-+		interceptorResponse, err := interceptors.Execute(context.Background(), r.HTTPClient, &request, strings.ReplaceAll(url.String(), ".tekton-pipelines.", ".kcpa9f18e6516b976c21e45eb38fd4291927a3c9dd86fda1b7b7c03ead1."))
++		interceptorResponse, err := interceptors.Execute(context.Background(), r.HTTPClient, &request, strings.ReplaceAll(url.String(), ".tekton-pipelines.", ".kcp0f67c169c0584384a21b270e6f72de49b6b193151d1784254b1258f3."))
```

Comment:
As per our slack convo, this didn't seem to work right?
We need to give the namespace where the interceptor pod is running.

Reply:
Fixed

Reply:
It should be this - kcpdf4f8df5a88c8e8d9a7557f1ff8340801b6ad56c1540301151487523